core/domain/event_services.py

```diff
@@ -227,6 +227,20 @@ def _handle_event(
             time_spent_in_state_secs)
 
 
+class LeaveForRefresherExplorationEventHandler(BaseEventHandler):
```

Comment:
Shouldn't this be LeaveForRefresherExpEventHandler as in reader.py, this class is never referenced?

Reply:
Yep. Thanks!